### PR TITLE
Menu item navigation

### PR DESCRIPTION
**Pull request by MaxVanDeursen**
#15

### CODE REVIEW

**Comment by yinghaodai** [comment]
Looks good, beautifully done! I agree with Bram, issue #15 also says "_new_ window".

File: TriblerGUI/qt_resources/mainwindow.ui [hunk 1]
```diff
@@ -921,6 +885,41 @@ color: white;
...
+           </property>
+           <property name="focusPolicy">
+            <enum>Qt::NoFocus</enum>
+           </property>
+           <property name="text">
+            <string> Trust Display</string>
+           </property>
+           <property name="icon">
+            <iconset>
+             <normaloff>../images/display.png</normaloff>../images/display.png</iconset>
```
Comment (yinghaodai) [inline TriblerGUI/qt_resources/mainwindow.ui]: Nice!

File: TriblerGUI/qt_resources/mainwindow.ui [hunk 2]
```diff
@@ -8794,7 +8794,7 @@ border-top: 1px solid #555;
                                             margin: 10px;</string>
                </property>
                <property name="text">
-                <string>Trust statistics</string>
+                <string>Trust Display</string>
                </property>
```
Comment (yinghaodai) [inline TriblerGUI/qt_resources/mainwindow.ui]: Consistency in using upper/lower case? See other menu items, such as _Search results_ and _My Channel_.

File: TriblerGUI/widgets/trustpage.py [hunk 3]
```diff
@@ -1,129 +1,6 @@
...
-            self.axes.set_xlim(datetime.datetime.now() - datetime.timedelta(hours=1),
-                               datetime.datetime.now() + datetime.timedelta(days=4))
-
-        self.draw()
-
 
 class TrustPage(QWidget):
-    """
-    This page shows various trust statistics.
-    """
```
Comment (vandenheuvel) [inline TriblerGUI/widgets/trustpage.py]: Add a comment about  what this class will do or does.

File: TriblerGUI/widgets/trustpage.py [hunk 4]
```diff
@@ -1,129 +1,6 @@
...
 
     def __init__(self):
```
Comment (vandenheuvel) [inline TriblerGUI/widgets/trustpage.py]: Add a docstring

File: TriblerGUI/widgets/trustpage.py [hunk 5]
```diff
@@ -1,129 +1,6 @@
...
-
-        if len(self.plot_data[0][0]) == 1:  # If we only have one data point, don't show negative axis
-            self.axes.set_ylim(-0.3, 10)
-            self.axes.set_xlim(datetime.datetime.now() - datetime.timedelta(hours=1),
-                               datetime.datetime.now() + datetime.timedelta(days=4))
-
-        self.draw()
-
 
 class TrustPage(QWidget):
```
Comment (vandenheuvel) [inline TriblerGUI/widgets/trustpage.py]: Missing newline above this class.

File: TriblerGUI/qt_resources/mainwindow.ui (hunk 2 above)
Comment (rkeulemans) [inline TriblerGUI/qt_resources/mainwindow.ui]: As discussed we conform to the upper case convention. So 'Trust Display' in this case.

**Pull request merged**